docs/examples/generatedCanonicalFields.json

```diff
@@ -0,0 +1,822 @@
+  },
+  {
+    "label": "Org Email",
+    "shortCode": "Org Email",
+    "dataType": "string"
+  },
+  {
+    "label": "Org DBA Name",
+    "shortCode": "Org DBA Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Banking Information",
+    "shortCode": "Org Banking Information",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Legal Docs",
+    "shortCode": "Org Legal Docs",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Board Members",
+    "shortCode": "Org Board Members",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Administrative Lead",
+    "shortCode": "Org Administrative Lead",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Annual Operating Budget",
+    "shortCode": "Org Annual Operating Budget",
+    "dataType": "number"
+  },
+  {
+    "label": "Org Lobbying Activities",
+    "shortCode": "Org Lobbying Activities",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Start Date",
+    "shortCode": "Org Start Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Geographic Areas",
+    "shortCode": "Org Geographic Areas",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Audited Financial Statements",
+    "shortCode": "Org Audited Financial Statements",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Grant Agreement Signatory",
+    "shortCode": "Org Grant Agreement Signatory",
+    "dataType": "string"
+  },
+  {
+    "label": "Org Fiscal Year End Date",
+    "shortCode": "Org Fiscal Year End Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Name(s)",
+    "shortCode": "Proposal Name(s)",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Contact Name",
+    "shortCode": "Proposal Primary Contact Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Email",
+    "shortCode": "Proposal Primary Email",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Primary Phone Number",
+    "shortCode": "Proposal Primary Phone Number",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Summary",
+    "shortCode": "Proposal Summary",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Budget",
+    "shortCode": "Proposal Budget",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Investment Start Date/Duration",
+    "shortCode": "Proposal Investment Start Date/Duration",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Investment End Date",
+    "shortCode": "Proposal Investment End Date",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Project Total Budget",
+    "shortCode": "Proposal Project Total Budget",
+    "dataType": "number"
+  },
+  {
+    "label": "Proposal Type",
+    "shortCode": "Proposal Type",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Fiscal Sponsorship Org Name",
+    "shortCode": "Proposal Fiscal Sponsorship Org Name",
+    "dataType": "string"
+  },
+  {
+    "label": "Proposal Location of Work (free-form string)",
```

Comment:
This one looks out of place. The parenthetical note in here looks it is in the upstream spreadsheets.

Reply:
Ah ha! Our first address!  The big kahuna!  This is the moment we must ask ourselves: are field transformations part of the pilot.

If the answer is "no" then we face a choice: do we store atomic address components as separate fields and put the burden of parsing / collating on the GMS's that collect addresses as free form strings, or do we store full address strings as single fields and put the burden of parsing / collating on the GMS that collect addresses as atomic strings.

I think this might warrant a separate issue / discussion.

One approach could be saying "we store EVERYTHING in non-atomic pieces but will eventually provide API utilities for parsing out atoms".  The logic here being that it's trivial for GMS's to construct a string from atoms; it's harder for GMS's to parse atoms from a string.

This also applies to Full Name vs First / Last name.

Reply:
I agree this should be a separate issue or discussion.